### ISSUE TITLE
The colored areas disappear every time the map reloads

### ISSUE THREAD

**Issue by FoffoDux83** [webbukkit/dynmap#3519]
**Issue Description:** *I have a problem with areas on the map. Every day the "marker_world.json" file resets itself automatically. I don't know what this is due to. It didn't happen before. The markers remain but the colored areas disappear. We use version 3.2 beta 3 forge 1.16.5.
Having to recreate all the zones on our public server would be too much work. It seems strange to me that since there is a file for this it cannot remain in memory.
Is there any step we can control or change?*

* **Dynmap Version:** *3.2 beta 3*
* **Server Version:** *forge 1.16.5.*
* **Server Host (if applicable):** *https://mc.ggservers.com/*

**Comment by FedUpWith-Tech** [comment]
The file gets reset if the default marker set or spawn marker gets deleted, are you deleting either of those things?

**Comment by FoffoDux83** [comment]
I think an operation like that was done, yes. We manage the server between several people and some time ago it was reset or something like that. But having the marker_world.json file we thought it worked.

Yesterday, by all means, I discovered the marker.yml file and in this file only the last zone I created is present. And it is the only area that appears when it recharges every day.
So I gave it a try: I inserted into this file, with the same format present inside and it actually works. Today, instead of having only one zone, I have three. So I think this is the file in question that needs to be changed and consequently the marker_worl.json file is generated as well. 
It is the only explanation and solution I have found on my own. 

Could it be considered as THE SOLUTION?
